litmus-portal/graphql-server/go.mod

```diff
@@ -6,11 +6,12 @@ require (
 	github.com/99designs/gqlgen v0.11.3
 	github.com/argoproj/argo-workflows/v3 v3.3.1
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
+	github.com/gin-contrib/cors v1.3.1
+	github.com/gin-gonic/gin v1.7.7
```

Comment:
1.7.7 is release in 2021. Can we use the latest one? If no major code change is required

Reply:
@imrajdas  let's take it in a different PR.

Reply:
[1.9.0](https://github.com/gin-gonic/gin/releases) is the latest version.
But higher than 1.7.7, We need to upgrade the version of Go 1.17.
So I just use that version like the[ authentication server in litmusChaos](https://github.com/litmuschaos/litmus/blob/ae9aeb2c76a8c5483184815801dc5ef0fb7616b4/litmus-portal/authentication/go.mod#L8)

<img width="1134" alt="스크린샷 2023-04-07 오후 9 25 40" src="https://user-images.githubusercontent.com/53862866/230608839-4680c039-14df-4f9c-a986-1d364d033b91.png">

Reply:
okay, lets do it in other PR